### PR TITLE
FOPTS-1327 Turbonomic Rightsize Volumes (AWS)

### PR DESCRIPTION
**Pull request by Rangelmv23**
### Description

AWS Rightsize Volumes policy needs to show the following information: current and new storage access IOPs, current and new storage amount, VM volume is attached to, action nature: disruptive or not, reversible or not, current and new throughput capacity, IOPs utilization, disk utilization, throughput utilization, system details URL.

### Issues Resolved

Added all requested fields and normalized incident fields.

### Link to Example Applied Policy

- AWS Turbonomic Rightsize Volumes: https://app.flexeratest.com/orgs/1105/automation/applied-policies/projects/60073?policyId=64d16397cd41360001891c7f

### Contribution Check List

- [x] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/turbonomics/rightsize_virtual_volumes_recommendations/aws/turbonomics_rightsize_virtual_volumes_recommendations.pt [hunk 1]
```diff
@@ -310,6 +461,69 @@ policy "pol_turbonomic_rightsize_volumes" do
...
+      end
+      field "size" do
+        label "Current Size Capacity (GB)"
+        path "currentSizeCapacity"
+      end
+      field "newSize" do
+        label "After Size Capacity (GB)"
+        path "afterSizeCapacity"
+      end
+      field "sizeUtilization" do
```
Comment (vadgun) [inline cost/turbonomics/rightsize_virtual_volumes_recommendations/aws/turbonomics_rightsize_virtual_volumes_recommendations.pt:500]: To be normalized according to excel normalization fields file, 
this Field can be renamed to : "newSizeUtil"

File: cost/turbonomics/rightsize_virtual_volumes_recommendations/aws/turbonomics_rightsize_virtual_volumes_recommendations.pt [hunk 2]
```diff
@@ -310,6 +461,69 @@ policy "pol_turbonomic_rightsize_volumes" do
...
+        label "Current Size Utilization (%)"
```
Comment (vadgun) [inline cost/turbonomics/rightsize_virtual_volumes_recommendations/aws/turbonomics_rightsize_virtual_volumes_recommendations.pt:501]: "Current" word in all labels need to be removed, 
Change "Size" by "Storage" in all labels.
E.g. "Storage % Utilization"

File: cost/turbonomics/rightsize_virtual_volumes_recommendations/aws/turbonomics_rightsize_virtual_volumes_recommendations.pt [hunk 3]
```diff
@@ -310,6 +461,69 @@ policy "pol_turbonomic_rightsize_volumes" do
...
+      field "iopsUtilization" do
+        label "Current IOPS Utilization (%)"
+        path "currentIopsUtilization"
+      end
+      field "newIopsUtilization" do
+        label "After IOPS Utilization (%)"
+        path "afterIopsUtilization"
+      end
+      field "size" do
+        label "Current Size Capacity (GB)"
```
Comment (vadgun) [inline cost/turbonomics/rightsize_virtual_volumes_recommendations/aws/turbonomics_rightsize_virtual_volumes_recommendations.pt:493]: Rename "Size" by "Storage"
Remove Current
Remove Capacity
E.g. "Storage (GB)"

File: cost/turbonomics/rightsize_virtual_volumes_recommendations/aws/turbonomics_rightsize_virtual_volumes_recommendations.pt [hunk 4]
```diff
@@ -310,6 +461,69 @@ policy "pol_turbonomic_rightsize_volumes" do
...
+        path "afterThroughputCapacity"
+      end
+      field "throughputUtilization" do
+        label "Current Throughput Utilization (%)"
+        path "currentThroughputUtilization"
+      end
+      field "newThroughputUtilization" do
+        label "After Throughput Utilization (%)"
+        path "afterThroughputUtilization"
+      end
```
Comment (vadgun) [inline cost/turbonomics/rightsize_virtual_volumes_recommendations/aws/turbonomics_rightsize_virtual_volumes_recommendations.pt:523]: ****

**Review by vadgun — CHANGES_REQUESTED** [review]

File: cost/turbonomics/rightsize_virtual_volumes_recommendations/aws/turbonomics_rightsize_virtual_volumes_recommendations.pt (hunk 4 above)
Comment (vadgun) [reply cost/turbonomics/rightsize_virtual_volumes_recommendations/aws/turbonomics_rightsize_virtual_volumes_recommendations.pt:523]: ****      field "throughput" do
        label "Throughput (MB/s)"
      end
      field "throughputP95" do
        label "Throughput % Utilization"
      end
      field "newThroughput" do
        label "New Throughput (MB/s)"
      end
      field "newThroughputP95" do
        label "New Throughput % Utilization"
      end

**Review by vadgun — CHANGES_REQUESTED** [review]

**Review by vadgun — COMMENTED** [review]
Looks good to me

**Review by XOmniverse — APPROVED** [review]
LGTM

**Review by vadgun — DISMISSED** [review]
Looks Good to me

**Review by night-codes — APPROVED** [review]

**Review by vadgun — APPROVED** [review]
Looks good to me

**Review by XOmniverse — APPROVED** [review]
LGTM

**Pull request merged**